src/mirror_prox.jl

```diff
@@ -70,13 +70,18 @@ struct MirrorProxParameters
   pock_chambolle_alpha::Union{Float64,Nothing}
 
   """
-  Used to bias the computation of the primal/dual balancing parameter
-  primal_weight. Must be positive. A value of 1 balances primal and dual
-  equally. Somewhat smaller values, e.g. 0.4, seem to yield better dual
-  solutions at the cost of increased primal infeasibility.
+  Used to bias the initial value of the primal/dual balancing parameter
+  primal_weight. Must be positive. See scale_invariant_initial_primal_weight.
   """
   primal_importance::Float64
 
+  """
+  If true, computes the initial primal weight with a scale-invariant formula
+  biased by primal_importance; see select_initial_primal_weight. If false,
```

Comment:
Perhaps "see select_initial_primal_weight() for more details" to make it clearer that this is pointing into the code, and not at another field or somewhere else in the documentation.